CONTRIBUTING.md

```diff
@@ -84,6 +84,22 @@ To contribute a patch, the workflow is as follows:
   1. Create topic branch
   1. Commit patches
 
+For GUI related issues or pull requests, the https://github.com/bitcoin-core/gui repository should be used.
```

Comment:
I think it may be worthwhile to state here from the start that the bitcoin-core/gui repo master branch is identical to the master branch in bitcoin/bitcoin. This may avoid some confusion while reading the rest of the section.